packages/components/src/components/Slider/__tests__/Slider.ct.tsx

```diff
@@ -0,0 +1,227 @@
+    const endingXPos = startingXPos + travelDistance;
+    const startingYPos = sliderBox.y + sliderBox.height / 2;
+
+    await slider.hover();
+    await page.mouse.down();
+    await page.mouse.move(endingXPos, startingYPos);
+    onBeforeMouseUp && onBeforeMouseUp();
+    await page.mouse.up();
+}
+
+test('should render a slider', async ({ mount }) => {
+    const component = await mount(<Slider data-test-id={SLIDER_TEST_ID} aria-label={ARIA_LABEL} defaultValue={[0]} />);
+    await expect(component).toBeVisible();
+    await expect(component.getByRole('slider')).toBeVisible();
+});
+
+test('should render two sliders', async ({ mount }) => {
+    const component = await mount(
+        <Slider data-test-id={SLIDER_TEST_ID} aria-label={ARIA_LABEL} defaultValue={[33, 67]} />,
+    );
+    await expect(component).toBeVisible();
+    const sliders = component.getByRole('slider');
+    const allSliders = await sliders.all();
+    for (const slider of allSliders) {
+        await expect(slider).toBeVisible();
+        await expect(slider).toHaveAttribute('aria-valuenow');
+    }
+    expect(allSliders).toHaveLength(2);
+});
+
+test('should render with default value', async ({ mount }) => {
+    const component = await mount(<Slider aria-label={ARIA_LABEL} defaultValue={[50]} />);
+    const slider = component.getByRole('slider');
+    expect(await slider.getAttribute('aria-valuenow')).toBe('50');
+});
+
+test('should render with controlled value', async ({ mount }) => {
+    const component = await mount(<Slider aria-label={ARIA_LABEL} value={[20]} />);
+    const slider = component.getByRole('slider');
+    expect(await slider.getAttribute('aria-valuenow')).toBe('20');
+});
+
+test('should render with custom data-test-id', async ({ mount }) => {
+    const component = await mount(<Slider data-test-id="custom-slider" aria-label={ARIA_LABEL} defaultValue={[0]} />);
+    expect(await component.getAttribute('data-test-id')).toBe('custom-slider');
+});
+
+test('should render with custom aria-label', async ({ mount }) => {
+    const component = await mount(
+        <Slider data-test-id={SLIDER_TEST_ID} aria-label="custom-label" defaultValue={[0]} />,
+    );
+    expect(await component.getAttribute('aria-label')).toBe('custom-label');
+});
+
+test('should render with custom aria-labelledby', async ({ mount }) => {
+    const component = await mount(
+        <Slider data-test-id={SLIDER_TEST_ID} aria-labelledby="custom-label" defaultValue={[0]} />,
+    );
+    const ariaLabelledBy = await component.getAttribute('aria-labelledby');
+    expect(ariaLabelledBy).toBe('custom-label');
+});
+
+test('should render with custom aria-describedby', async ({ mount }) => {
+    const component = await mount(
+        <Slider aria-label="custom-aria" aria-describedby="custom-description" defaultValue={[0]} />,
+    );
+    const ariaDescribedBy = await component.getAttribute('aria-describedby');
+    expect(ariaDescribedBy).toBe('custom-description');
+});
```

Comment:
Anyway to merge them instead of separated cases?

Reply:
can do